test/test_integration.py

```diff
@@ -361,7 +361,7 @@
 xml_test_parameters = [(test, test['expect_out'])
                        for test in tests if test != csv_latin1_encoding]
 
-file_location = os.path.dirname(os.path.realpath(__file__))
+file_location = os.path.dirname(os.path.realpath(_file))
```

Comment:
Shouldn't this be `os.path.dirname(os.path.realpath(__file__))`?

Reply:
In line 62 of compile.py I think the keyword for file was changed from __file__ to _file. I wasn't sure if that was a new change or an outdated one, but if we go with __file__, I can quickly change that over!